R/content.R

```diff
@@ -1637,7 +1522,13 @@ get_content_packages <- function(content) {
 #' library(connectapi)
 #' client <- connect()
 #'
-#' search_content(client, q = "")
+#' my_content <- search_content(client, q = "owner:@me")
+#'
+#' shiny_content <- purrr::keep(my_content, function(x) {
+#'   x$content$app_mode == "rmd-shiny"
+#' })
```

Comment:
This is a bit of an odd example, no? I suspect it's there to show that (currently) one should use `purrr::keep(...)` for filtering, but in this case, you would _actually_ want to do `my_content <- search_content(client, q = "owner:@me type:rmd-shiny")` to do this. 

There's an arc of work to make the list more-easily-filterable that we should take up (later/next), but IMO we should document the stuff we're building now with how it should be used today, even if there might be other ways to do / other tasks folks might do with stuff like this in the future.

Reply:
Good point!! Perhaps I should use the longer example I used in GitHub PRs that uses a comparison that isn't possible in the filter API at the moment.

[edit] And/or just remove this example.